### ISSUE TITLE
Does not work with kaggle data.

### ISSUE THREAD

**Issue by ignvinay** [98mprice/pyAudioClassification#2]
Hi micah5,
     Its great work for developing classification based on audio.
I tried your code, but seem to not work properly.
I tried training and testing with kaggle data, which you can also try 
https://www.kaggle.com/mmoreaux/audio-cats-and-dogs/version/5#cats_dogs.zip

It seem to be dropping one label and also misclassifying all the time. I increased epochs and got close to 90% accuracy, but still then seem to do wrongly. Even test data given is training data , that also seem to not work.

Sample output :

(tf_gpu) d:\Vinay\Audio_classification\pyAudioClassification\example>python test.py
Using TensorFlow backend.
2019-04-15 15:53:10.947087: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2019-04-15 15:53:11.788533: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1432] Found device 0 with properties:
name: GeForce GTX 1050 major: 6 minor: 1 memoryClockRate(GHz): 1.493
pciBusID: 0000:01:00.0
totalMemory: 4.00GiB freeMemory: 3.30GiB
2019-04-15 15:53:11.792768: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1511] Adding visible gpu devices: 0
2019-04-15 15:53:12.162764: I tensorflow/core/common_runtime/gpu/gpu_device.cc:982] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-04-15 15:53:12.172438: I tensorflow/core/common_runtime/gpu/gpu_device.cc:988]      0
2019-04-15 15:53:12.173703: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 0:   N
2019-04-15 15:53:12.175952: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 3018 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1050, pci bus id: 0000:01:00.0, compute capability: 6.1)
Lets predict the file  data/cat/cat_1.wav
1. dog 0.0% (index 0)
[[6.274987e-21]]
